recipes/hyperspy_gui_ipywidgets/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Please drop this file as we don't need it in the recipe since it is in the source.